canonical/ght
# add alternative login method

## Done

Add an option, `--new` that you can pass with any command. This will trigger an alternative login flow that uses puppeteer to go through the login instead of the current implementation, which is more low level.

This is an attempt to solve authentication issues for some users.

If this works and doesn't cause any side effect, it can later become the standard behaviour.

## QA

```
yarn dev login --new
```

Try to run a command and check that it works:

```
yarn dev replicate -i --new
```

Try to run a command without `--new` and check that it runs too:

```
yarn dev replicate -i
```

## Issue

https://warthogs.atlassian.net/browse/WD-6864
